Tests/NumLib/ODEs.h

```diff
@@ -92,7 +92,8 @@ class ODETraits<ODE1>
 
 const double ODETraits<ODE1>::t0 = 0.0;
 
-const double ODETraits<ODE1>::t_end = 30.0;
+const double ODETraits<ODE1>::t_end =
+    2.0 * 3.141592653589793238462643383279502884197169399375105820974944;
```

Comment:
This is true analytically, but this approach might suffer from numerical inaccuracies of the atan() function. But in the future we should introduce a global constant for it. @norihiro-w uses `M_PI` in some current PR. But AFAIK that's a non-standard macro.